modules/gdscript/gdscript.cpp

```diff
@@ -152,12 +152,13 @@ Variant GDScript::_new(const Variant **p_args, int p_argcount, Variant::CallErro
 	}
 
 	ERR_FAIL_COND_V(_baseptr->native.is_null(), Variant());
-
 	if (_baseptr->native.ptr()) {
 		owner = _baseptr->native->instance();
 	} else {
-		owner = memnew(Reference); //by default, no base means use reference
```

Comment:
Any particular reason to remove this comment?

Reply:
Bump.